src/services/schedule.ts

```diff
@@ -67,19 +77,29 @@ function makeScheduleCourses(schedule: CourseSchedule[]): ScheduleEvent[] {
       .set('y', today.year())
       .set('M', today.month())
       .set('d', today.day());
-    for (const day of getMeetingDays(course)) {
-      const courseEvent = {
-        id: eventId,
+
+    const meetingDays = getMeetingDays(course);
+
+    if (course.ROOM_CDE === 'ASY') {
+      return summerMeetingDays.map((day) => ({
```

Comment:
It's not just summer courses that are ASY.  There are also some during other terms.  Please rename summerMeetingDays.  (Maybe "asyncMeetingDays"?)  I haven't looked at all the uses, so don't blindly take my suggestions.

Reply:
This is a good point, we changed the variable name and re-pushed it.